docs/setup/deploy-edge-node.md

```diff
@@ -0,0 +1,43 @@
+# Deploy Edge Node Manually
```

Comment:
How to jump here from setup.md? I'd suggest to keep it in setup.md. For example:

#### Deploy the edge node
Edge node can be registered automatically.
##### Deploy Edge Node Manually
If you want to deploy edge node manually, here is an example.

Reply:
please help modify it more clear，like: L5-L16, we can just delete it.